stl/inc/functional

```diff
@@ -795,7 +795,8 @@ private:
 constexpr size_t _Space_size = (_Small_object_num_ptrs - 1) * sizeof(void*);
 
 template <class _Impl> // determine whether _Impl must be dynamically allocated
-_INLINE_VAR constexpr bool _Is_large = (_Space_size < sizeof(_Impl)) || !_Impl::_Nothrow_move::value;
+_INLINE_VAR constexpr bool _Is_large = (_Space_size < sizeof(_Impl)) || alignof(_Impl) > alignof(max_align_t)

```

Comment:
Stylistic nitpick: the code is now inconsistent about less-than vs. greater-than, and it had pre-existing extra parentheses which are inconsistently missing from the newly added code (these parentheses were a relic from when these were template arguments instead of initializing a variable template, as I recall). So, I recommend:
```suggestion
_INLINE_VAR constexpr bool _Is_large = sizeof(_Impl) > _Space_size || alignof(_Impl) > alignof(max_align_t)
```
This reads more naturally: if the impl is too big, or the impl is too highly aligned, or meow.